src/main/java/uk/gov/justice/digital/delius/data/api/CreatePrisonOffenderManager.java

```diff
@@ -23,39 +23,36 @@
     // This annotation appears to be ignored by swagger docs and nobody seems to care: https://github.com/springfox/springfox/issues/2237
     @ApiModelProperty(value = "Name of offender manager. If passed then must contain both forename(s) and surname", example = "officer: {\"forenames\": \"John\", \"surname\": \"Smith\" }")
     private Human officer;
-    @ApiModelProperty(value = "Officer staff code. If not present officer will be used to lookup staff member", example = "N07A001")
-    private String officerCode;
+    @ApiModelProperty(value = "Officer staff ID. If not present officer will be used to lookup staff member", example = "1234567")
+    private Long staffId;
     @ApiModelProperty(value = "Prison institution code in NOMIS", required = true, example = "MDI")
     private String nomsPrisonInstitutionCode;
 
     /**
      * This is tested in the API @see OffenderResources_AllocatePrisonOffenderManagerAPITest
      */
     public Optional<String> validate() {
-        final var prisonCodeMissing = isNullOrEmpty(getNomsPrisonInstitutionCode());
-        final var officerCodeExists = !isNullOrEmpty(getOfficerCode());
-        final var officerExists = !isNull(getOfficer());
```

Comment:
lang3 seems to have a nicer API so can do an `isNotEmpty` here rather than having to use `!`